mkdocs.yml

```diff
@@ -102,6 +102,7 @@ nav:
         - reference/cpp/index.md
       - Differences: reference/cpp/differences.md
       - Tools: reference/cpp/tools.md
+      - NET Workspace Structure: reference/cpp/net_workspace.md
```

Comment:
- New project tab will be introduced in #93, wait until that is merged into main before merging main into this branch so that you can update the page tree accordingly